### PR TITLE
add table for cost by region monthly

### PR DESCRIPTION
**Pull request by razbne**
# Integration test logs
<details>
  <summary>Logs</summary>

```
N/A
```
</details>

# Example query results
<details>
  <summary>Results</summary>

```
with ranked_costs as (
  select
    region,
    period_start,
    unblended_cost_amount :: numeric :: money,
    rank() over(
      partition by region
      order by
        unblended_cost_amount desc
    )
  from
    aws_cost_by_region_monthly
)
select
  *
from
  ranked_costs
where
  rank = 1;
+----------------+---------------------------+-----------------------+------+
| region         | period_start              | unblended_cost_amount | rank |
+----------------+---------------------------+-----------------------+------+
| NoRegion       | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| NoRegion       | 2024-08-01T03:00:00+03:00 | $0.00                 | 1    |
| ap-northeast-1 | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| ap-northeast-2 | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| ap-northeast-3 | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| ap-south-1     | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| ap-southeast-1 | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| ap-southeast-2 | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| ca-central-1   | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| eu-central-1   | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| eu-north-1     | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| eu-west-1      | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| eu-west-2      | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| eu-west-3      | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| global         | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| sa-east-1      | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| us-east-1      | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| us-east-2      | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| us-west-1      | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| us-west-2      | 2024-09-01T03:00:00+03:00 | $0.00                 | 1    |
| <null>         | 2023-10-01T03:00:00+03:00 | $0.00                 | 1    |
| <null>         | 2024-04-01T03:00:00+03:00 | $0.00                 | 1    |
| <null>         | 2024-07-01T03:00:00+03:00 | $0.00                 | 1    |
| <null>         | 2024-02-01T02:00:00+02:00 | $0.00                 | 1    |
| <null>         | 2023-09-23T03:00:00+03:00 | $0.00                 | 1    |
| <null>         | 2024-06-01T03:00:00+03:00 | $0.00                 | 1    |
| <null>         | 2024-05-01T03:00:00+03:00 | $0.00                 | 1    |
| <null>         | 2024-03-01T02:00:00+02:00 | $0.00                 | 1    |
| <null>         | 2023-11-01T02:00:00+02:00 | $0.00                 | 1    |
| <null>         | 2023-12-01T02:00:00+02:00 | $0.00                 | 1    |
| <null>         | 2024-01-01T02:00:00+02:00 | $0.00                 | 1    |
+----------------+---------------------------+-----------------------+------+
```

### CODE REVIEW

File: docs/tables/aws_cost_by_region_monthly.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+from 
+  aws_cost_by_region_monthly
+group by
+  region
+order by
+  average desc
+limit 10;
+```
+
+
```
Comment (ParthaI) [inline docs/tables/aws_cost_by_region_monthly.md:121]:
```suggestion
```

File: docs/tables/aws_cost_by_region_monthly.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+from 
+  aws_cost_by_region_monthly
+group by
+  region
+order by
+  sum desc
+limit 10;
+```
+
+
```
Comment (ParthaI) [inline docs/tables/aws_cost_by_region_monthly.md:153]:
```suggestion
```

**Review by ParthaI — COMMENTED** [review]
Hello @razbne, left a few review comments, please take look, thanks!

**Comment by misraved** [comment]
Hi @razbne, did you get a chance to work on the review comments suggested by @ParthaI?

Please feel free to reach out if you have questions 👍!

Happy to help 😄!!

**Pull request merged**